tests/unit/ml_handlers/test_auto_ts.py

```diff
@@ -0,0 +1,154 @@
+import time
+
+import pandas as pd
+import pytest
+from mindsdb_sql import parse_sql
+from ..executor_test_base import BaseExecutorTest
+
+
+class TestAuto_ts(BaseExecutorTest):
```

Comment:
More generally, once you fix the points previously mentioned, please make sure tests actually call the predictor with `JOIN` and `LATEST`. Refer to our docs for examples on how this looks like. That will ensure this is a properly implemented time series integration (as in, it conforms to how others behave, too).